src/plugins/vis_types/xy/public/config/get_axis.ts

```diff
@@ -50,9 +48,10 @@ export function getAxis<S extends XScaleType | YScaleType>(
     : {
         show: valueAxis === axis.id,
       };
-  // Date range formatter applied on xAccessor
-  const tickFormatter =
-    aggType === BUCKET_TYPES.DATE_RANGE || aggType === BUCKET_TYPES.RANGE ? identity : formatter;
+
+  const tickFormatter = (v: any) =>
```

Comment:
Just checking here? Can we not use `any` here?

Reply:
Done)